### PR TITLE
xmlsec1: adopting package at 1.2.32

### PR DESCRIPTION
**Pull request by Lazar-Kovacevic**
Please fill out the template.

Read: https://k1sslinux.org/wiki/kiss/style-guide

Please include the manifest for version bumps and new packages.

---

## Description of package
xmlsec1, xml security library, dependency of oath-toolkit

## Installed manifest of package

(`kiss manifest <pkg>`)

```
/var/db/kiss/installed/xmlsec1/version
/var/db/kiss/installed/xmlsec1/sources
/var/db/kiss/installed/xmlsec1/manifest
/var/db/kiss/installed/xmlsec1/depends
/var/db/kiss/installed/xmlsec1/checksums
/var/db/kiss/installed/xmlsec1/build
/var/db/kiss/installed/xmlsec1/
/var/db/kiss/installed/
/var/db/kiss/
/var/db/
/var/
/usr/share/man/man1/xmlsec1.1
/usr/share/man/man1/xmlsec1-config.1
/usr/share/man/man1/
/usr/share/man/
/usr/share/aclocal/xmlsec1.m4
/usr/share/aclocal/
/usr/share/
/usr/lib/xmlsec1Conf.sh
/usr/lib/pkgconfig/xmlsec1.pc
/usr/lib/pkgconfig/xmlsec1-openssl.pc
/usr/lib/pkgconfig/
/usr/lib/libxmlsec1.so.1.2.32
/usr/lib/libxmlsec1.so.1
/usr/lib/libxmlsec1.so
/usr/lib/libxmlsec1.a
/usr/lib/libxmlsec1-openssl.so.1.2.32
/usr/lib/libxmlsec1-openssl.so.1
/usr/lib/libxmlsec1-openssl.so
/usr/lib/libxmlsec1-openssl.a
/usr/lib/
/usr/include/xmlsec1/xmlsec/xmltree.h
/usr/include/xmlsec1/xmlsec/xmlsec.h
/usr/include/xmlsec1/xmlsec/xmlenc.h
/usr/include/xmlsec1/xmlsec/xmldsig.h
/usr/include/xmlsec1/xmlsec/x509.h
/usr/include/xmlsec1/xmlsec/version.h
/usr/include/xmlsec1/xmlsec/transforms.h
/usr/include/xmlsec1/xmlsec/templates.h
/usr/include/xmlsec1/xmlsec/strings.h
/usr/include/xmlsec1/xmlsec/soap.h
/usr/include/xmlsec1/xmlsec/private.h
/usr/include/xmlsec1/xmlsec/parser.h
/usr/include/xmlsec1/xmlsec/openssl/x509.h
/usr/include/xmlsec1/xmlsec/openssl/symbols.h
/usr/include/xmlsec1/xmlsec/openssl/evp.h
/usr/include/xmlsec1/xmlsec/openssl/crypto.h
/usr/include/xmlsec1/xmlsec/openssl/bn.h
/usr/include/xmlsec1/xmlsec/openssl/app.h
/usr/include/xmlsec1/xmlsec/openssl/
/usr/include/xmlsec1/xmlsec/nodeset.h
/usr/include/xmlsec1/xmlsec/membuf.h
/usr/include/xmlsec1/xmlsec/list.h
/usr/include/xmlsec1/xmlsec/keysmngr.h
/usr/include/xmlsec1/xmlsec/keysdata.h
/usr/include/xmlsec1/xmlsec/keys.h
/usr/include/xmlsec1/xmlsec/keyinfo.h
/usr/include/xmlsec1/xmlsec/io.h
/usr/include/xmlsec1/xmlsec/exports.h
/usr/include/xmlsec1/xmlsec/errors.h
/usr/include/xmlsec1/xmlsec/dl.h
/usr/include/xmlsec1/xmlsec/crypto.h
/usr/include/xmlsec1/xmlsec/buffer.h
/usr/include/xmlsec1/xmlsec/bn.h
/usr/include/xmlsec1/xmlsec/base64.h
/usr/include/xmlsec1/xmlsec/app.h
/usr/include/xmlsec1/xmlsec/
/usr/include/xmlsec1/
/usr/include/
/usr/bin/xmlsec1-config
/usr/bin/xmlsec1
/usr/bin/
/usr/
```

## New package

- [ ] Latest upstream version.
- [ ] I agree to maintain this package.
- [ ] I agree to notify the @kiss-community maintainers if I can no longer maintain this package.

## Existing package

- [ ] I am the maintainer of this package.

### CODE REVIEW

**Comment by dilyn-corner** [comment]
You could also take `oath-toolkit` if you'd like!

File: community/xmlsec1/build [hunk 1]
```diff
@@ -1,12 +1,33 @@
...
+  --enable-x509=yes \
+  --enable-des=yes \
+  --enable-aes=yes \
+  --enable-gost=yes \
+  --enable-xmldsig=yes \
+  --enable-xmlenc=yes \
+  --enable-soap=yes \
+  --enable-mans=yes \
+  --enable-docs=no \
+  --enable-apps=yes
```
Comment (dilyn-corner) [inline community/xmlsec1/build:30]: Do... Do these have to be explicit? Seems like most of this should be covered by `--with-default-crypto=openssl`

File: community/xmlsec1/build [hunk 2]
```diff
@@ -1,12 +1,33 @@
...
 ./configure \
-    --prefix=/usr \
-    --with-default-crypto=openssl \
-    --without-gcrypt \
-    --without-gnutls \
-    --without-nss \
-    --without-libxslt
+  --prefix=/usr \
+  --bindir=/usr/bin \
+  --sbindir=/usr/sbin \
```
Comment (dilyn-corner) [inline community/xmlsec1/build:6]: `/usr/sbin` -> `/usr/bin` ?

File: community/xmlsec1/build (hunk 1 above)
Comment (Lazar-Kovacevic) [reply community/xmlsec1/build]: To be honest, i don't have a clue, i just copied from ./configure --help in hope to enable all functionality, but i have updated the package and erased all of that, lets just trust the defaults, i don't see anything obvious missing.

**Pull request merged**